frontend/src/services/applicationService.tsx

```diff
@@ -0,0 +1,11 @@
+import api from '../common/helpers/apiHelper';
+
+export const addApp = async (appName: string) => {
+  const response = await api.post('/api/applications', { appName });
+  return response;
+};
```

Comment:
Let's keep service functions clearer - no need to call API method with await and then return result, you can just do it like this: 
```suggestion
export const addApp = (appName: string) => api.post('/api/applications', { appName });
```
Also, API methods accept generic, so you always need to set a type of data returned from server. Like this: `api.post<IApp>(...)`